recipe/meta.yaml

```diff
@@ -10,9 +10,10 @@ source:
   sha256: 207b8951e4623bb2e87e0a3b3014ae12b7c4a696e7377b1910d9978dd1ad28eb
 
 build:
-  # We skip the build on Windows because of linkage problems with libprotobuf.
-  # See: https://github.com/man-group/ArcticDB/pull/449
-  skip: true  # [win]
```

Comment:
We need to keep the Windows skip, don't we? cf main https://github.com/conda-forge/arcticdb-feedstock/blob/main/recipe/meta.yaml#L15